app/javascript/spec/report-data-table.spec.js

```diff
@@ -31,10 +31,10 @@ describe('<ReportDataTable />', () => {
     const limit = 20;
     const offset = 0;
     return `/api/results/${initialProps.reportResultId}?\
-expand_value_format=true&\
-hash_attribute=result_set&\
-sort_by=${sortBy}&sort_order=${sortDirection}&\
-limit=${limit}&offset=${offset}`;
+    expand_value_format=true&\
+    hash_attribute=result_set&\
+    sort_by=${sortBy}&sort_order=${sortDirection}&\
+    limit=${limit}&offset=${offset}`;
```

Comment:
Can you indent like this for something that will be a URL?  Not sure if backtick strings handle indenting.

Reply:
This is still a question as the indent is still present, even though it says Outdated.

Reply:
@Fryguy , an example of the current url - `"/api/results/5?\n    expand_value_format=true&\n    hash_attribute=result_set&\n    sort_by=''&\n    sort_order=asc&\n    limit=3&\n    offset=0"`

so, the url will append the ascii code for spaces and new line..

Reply:
@Fryguy , can we use like this

```
const url = new URL(`/api/results/${reportResultId}`);
url.searchParams.append('expand_value_format', 'true');
...
```